cluster/gce/config-test.sh

```diff
@@ -132,3 +132,5 @@ OPENCONTRAIL_PUBLIC_SUBNET="${OPENCONTRAIL_PUBLIC_SUBNET:-10.1.0.0/16}"
 
 # Optional: if set to true, kube-up will configure the cluster to run e2e tests.
 E2E_STORAGE_TEST_ENVIRONMENT=${KUBE_E2E_STORAGE_TEST_ENVIRONMENT:-false}
+# Overlay network settings
+OVERLAY_NETWORK=${OVERLAY_NETWORK:-true}
```

Comment:
I think we should probably attach to the NETWORK_PROVIDER= abstraction above.  Set the default to 'flannel' and allow "" "flannel" and "opencontrail" options (since those are the integrations we have at this level, so far)

what do you think?